Hymns.xcodeproj/project.pbxproj

```diff
@@ -24,6 +24,8 @@
 		5CA784A4242D1CF800B57903 /* HomeView.swift in Sources */ = {isa = PBXBuildFile; fileRef = 5CA784A3242D1CF800B57903 /* HomeView.swift */; };
 		5CAF7433242B98DF00A847E2 /* DummyTestData.swift in Sources */ = {isa = PBXBuildFile; fileRef = 5CAF7432242B98DF00A847E2 /* DummyTestData.swift */; };
 		5CAF7435242B991100A847E2 /* SearchBarView.swift in Sources */ = {isa = PBXBuildFile; fileRef = 5CAF7434242B991100A847E2 /* SearchBarView.swift */; };
+		B51BA5C92430075300CA52F7 /* DetailHymnScreen.swift in Sources */ = {isa = PBXBuildFile; fileRef = B51BA5C82430075300CA52F7 /* DetailHymnScreen.swift */; };
+		B51BA5CB2430076C00CA52F7 /* DetailHymnLyrics.swift in Sources */ = {isa = PBXBuildFile; fileRef = B51BA5CA2430076C00CA52F7 /* DetailHymnLyrics.swift */; };
```

Comment:
I forgot what we said, but this file needs to be changed every time right? Can we remove this from the version control? Since it's just generated code and isn't really user-readable. Or do we need to keep it in git?

Reply:
project.pbxproj is an important file in the Xcode configuration bundle. It is responsible for maintaining references to all of the linked files and their groupings, linked frameworks, and most importantly, the project’s build settings. Because of this, we **cannot** exclude project.pbxproj from version control. Earlier when I excluded it from a git commit then xcode would not even recognize that new file when it was pull from github.